src/libraries/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.Windows.cs

```diff
@@ -429,7 +424,7 @@ private int InternalValueCountCore()
         {
             int values = 0;
             int junk = 0;
-            int ret = Interop.Advapi32.RegQueryInfoKey(_hkey,
+            int ret = Interop.Advapi32.RegQueryInfoKey(_hkey!,
```

Comment:
I guess junk can be a discard (`_`) if you are doing cleanup.

Reply:
Ok. There are some small things I can take care of:
A) To remove unneeded usings
B) To inline some variable declarations
C) To simplify some 'default' expressions

In total ~10 spot (is it the correct term in english?).

Which refactorings should I do?

Reply:
I think it's fine to do simple, safe localized tweaks to align with the common style in this repo and don't make the review harder. Sounds like those would.